cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -133,6 +133,7 @@ share
     paymentCred         ByteString Maybe    sqltype=hash28type
     stakeAddressId      StakeAddressId Maybe OnDeleteCascade
     value               DbLovelace          sqltype=lovelace
+    dataHash            ByteString Maybe
```

Comment:
This is a `Blake2b_256` hash, so we can use a `sqltype=hash32type`.

Reply:
Fixed, thanks!